### ISSUE TITLE
Boards Manager unreadable.

### ISSUE THREAD

**Issue by bjwest** [jeffThompson/DarkArduinoTheme#15]
The Boards Manager dialog is unreadable using this theme with a dark desktop theme.  The dialog still uses normal colors but the fonts use the desktop colors and are therefore unreadable.

**Comment by jeffThompson** [comment]
Dark desktop theme meaning your OS? If so, which OS are you using? Can you include a screenshot?

**Comment by jfdesignnet** [comment]
here is the lookie on kubuntu kde 5.11 with breeze-dark theme :
https://imgur.com/a/BePak
thanks for the share, appreciate it.

**Comment by 2114L3** [comment]
get the same on archlinux 4.17, xfce 4.12, adwaita-dark

**Comment by jeffThompson** [comment]
Ah ok, this makes more sense. I think it has to do with your native OS themes (both of which, I believe, are "dark" too). The Arduino IDE uses the native desktop colors for the Boards Manager but also grabs some parameters from the IDE theme. There are sadly no settings available in the theme file specifically for the Boards Manager.

TLDR: since your OS theme is dark, it conflicts in the Boards Manager with the IDE theme. I don't think there's a way to fix this.

Closing. The only way to get around this would be a pull request on the IDE source code allowing you to change the Boards Manager specifically. Sorry!

**Comment by comperem** [comment]
boo.

this is still a problem in 1.8.10.

disappointing thread. this is where the problems are supposed to be addressed not dismissed.
